tests/middleware/test_logging_middleware.py

```diff
@@ -190,3 +192,33 @@ async def hello_world_handler() -> Dict[str, str]:
         response = client.get("/")
         assert response.status_code == HTTP_200_OK
         assert len(caplog.messages) == 2
+
+
+def test_logging_middleware_with_session_middleware() -> None:
+    # https://github.com/starlite-api/starlite/issues/1228
+
+    @post("/")
+    async def set_session(request: Request) -> None:
+        request.set_session({"hello": "world"})
+
+    @get("/")
+    async def get_session() -> None:
+        pass
+
+    logging_middleware_config = LoggingMiddlewareConfig()
+    session_config = MemoryBackendConfig()
+
+    with create_test_client(
+        [set_session, get_session],
+        logging_config=LoggingConfig(),
+        middleware=[logging_middleware_config.middleware, session_config.middleware],
+    ) as client:
+        response = client.post("/")
+        assert response.status_code == HTTP_201_CREATED
+        assert "session" in client.cookies
+        assert len(client.cookies["session"]) == 64
```

Comment:
I think the better solution here would be to check against a constant, known value. The length based checking only works incidentally.

Reply:
I originally was just going to check for `client.cookies["session"] != "*****"` but I figured that had some room for error as well.

What if I did `client.cookies["session"] != "*****"` but also confirmed the session id did not change between the two requests:

```python
response = client.post("/")
assert response.status_code == HTTP_201_CREATED
assert "session" in client.cookies
assert client.cookies["session"] != "*****"
session_id = client.cookies["session"]

response = client.get("/")
assert response.status_code == HTTP_200_OK
assert "session" in client.cookies
assert client.cookies["session"] == session_id
```

Reply:
That's a reasonable approach

Reply:
change has been pushed